src/Repository/RedirectRepositoryInterface.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace Setono\SyliusRedirectPlugin\Repository;
+
+use Setono\SyliusRedirectPlugin\Model\RedirectInterface;
+use Sylius\Component\Resource\Repository\RepositoryInterface;
+
+interface RedirectRepositoryInterface extends RepositoryInterface
+{
+    /**
+     * @param string $source
+     *
+     * @return RedirectInterface|null
+     *
+     * @throws \Doctrine\ORM\NonUniqueResultException
```

Comment:
This `@throws` should probably only be in the concrete class

Reply:
Indeed